src/test/java/com/datasonnet/spi/DataFormatPluginsTest.java

```diff
@@ -13,7 +13,7 @@
 
     @Test
     void testDataFormatPlugins() throws Exception {
-        DataFormatService service = DataFormatService.getInstance();
+        DataFormatService service = new DataFormatService();
```

Comment:
The test failed before if tests registering the XML plugin had run before this. This isolates the test.